src/components/home/homeScreen/about.tsx

```diff
@@ -0,0 +1,35 @@
+import smiliesIcon from 'assets/images/smilies.png'
+
+import 'components/home/homeScreen/about.css'
+
+const about = () => {
```

Comment:
The convention for naming components is to use capital
```suggestion
const About = () => {
```